gcc11 build failure

It would seem that the headers have changed between gcc versions (as confirmed by Hurlitzer on discord) and an `#include <cstddef>` is missing in certain places.


Namely, `synthLib/audiobuffer.h`:

```
cd /home/user/Documents/Dev/gearmulator/temp/cmake_linux/source/synthLib && /usr/bin/c++ -DRELEASE -I/home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/.. -I/home/user/Documents/Dev/gearmulator/source/dsp56300/source/asmjit/src -msse -O3 -fPIC -pthread -DASMJIT_STATIC -std=gnu++17 -MD -MT source/synthLib/CMakeFiles/synthLib.dir/audiobuffer.cpp.o -MF CMakeFiles/synthLib.dir/audiobuffer.cpp.o.d -o CMakeFiles/synthLib.dir/audiobuffer.cpp.o -c /home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.cpp
In file included from /home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.cpp:1:
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:12:30: error: ‘size_t’ has not been declared
   12 |                 void reserve(size_t _capacity);
      |                              ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:13:29: error: ‘size_t’ has not been declared
   13 |                 void resize(size_t _capacity);
      |                             ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:15:44: error: ‘size_t’ has not been declared
   15 |                 void append(float** _data, size_t _size);
      |                                            ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:17:29: error: ‘size_t’ has not been declared
   17 |                 void remove(size_t _count);
      |                             ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:19:54: error: ‘size_t’ has not been declared
   19 |                 void fillPointers(float** _pointers, size_t _offset = 0);
      |                                                      ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:20:17: error: ‘size_t’ does not name a type
   20 |                 size_t size() const;
      |                 ^~~~~~
/home/user/Documents/Dev/gearmulator/source/synthLib/audiobuffer.h:3:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
    2 | #include <vector>
  +++ |+#include <cstddef>
    3 | 
```

And `opcodeinfo.h`, but that I will report to the other project in a second:

```
cd /home/user/Documents/Dev/gearmulator/temp/cmake_linux/source/dsp56300/source/dsp56kEmu && /usr/bin/c++ -DRELEASE -I/home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/.. -I/home/user/Documents/Dev/gearmulator/source/dsp56300/source/asmjit/src -msse -O3 -fPIC -pthread -DASMJIT_STATIC -std=gnu++17 -MD -MT source/dsp56300/source/dsp56kEmu/CMakeFiles/dsp56kEmu.dir/disasm.cpp.o -MF CMakeFiles/dsp56kEmu.dir/disasm.cpp.o.d -o CMakeFiles/dsp56kEmu.dir/disasm.cpp.o -c /home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/disasm.cpp
In file included from /home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/opcodes.h:6,
                 from /home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/disasm.cpp:1:
/home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/opcodeinfo.h: In function ‘constexpr uint32_t dsp56k::createMask(const char (&)[25], char, char)’:
/home/user/Documents/Dev/gearmulator/source/dsp56300/source/dsp56kEmu/opcodeinfo.h:14:21: error: ‘size_t’ was not declared in this scope; did you mean ‘std::size_t’?
   14 |                 for(size_t i=0; i<len; ++i)
      |                     ^~~~~~
      |                     std::size_t
```
No errors are reported when using gcc10 instead.
Adding includes in these two places makes the build pass on gcc11, and it still works on gcc10.

If that looks good to you I can submit a PR.

--- a/source/synthLib/audiobuffer.h
+++ b/source/synthLib/audiobuffer.h
@@ -1,4 +1,5 @@
 #pragma once
+#include <cstddef>
 #include <vector>
 
 namespace synthLib

